internal/pkg/pkg.go

```diff
@@ -158,8 +169,12 @@ func (p *Pkg) DirectSubpackages() ([]*Pkg, error) {
 	for _, subPkgPath := range packagePaths {
 		subPkg, err := New(filepath.Join(p.UniquePath.String(), subPkgPath))
```

Comment:
Instead of adjusting the DisplayPath after the fact, why not have constructor for Pkg (`New`) just set it properly by taking an argument to the top-level directory?

Reply:
This is needed only while initializing subpackages. And initializing subpackages logic is abstracted within pkg.go. So I feel it is better to keep New constructor simple, the way it is.

Reply:
Subpackage is a Pkg, so the it's less hacky for c'tor to do it. But I'm not going to block this PR on this given it's already long in the tooth.